app/controllers/sc_collections_controller.rb

```diff
@@ -257,6 +257,15 @@ def convert_manifest
       ocr_text = ocr ? 'and OCR text ' : ''
       flash[:notice] = t('.metadata_is_being_imported', ocr_text: ocr_text)
     end
+
+    # TODO: This patches CDM imports. It would be good to refactor it into interactor to make testing easier
+    Elasticsearch::Collection::SyncJob.perform_later(
```

Comment:
How will the `perform_later` interact with the rake task?  Do we run into a possibility of race conditions, where the rake task is still running after the SyncJob completes?